app/assets/scripts/components/indicators/indicator-shipping.js

```diff
@@ -0,0 +1,298 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Prose from '../../styles/type/prose';
+import Gridder from '../../styles/gridder';
+import InpageHGroup from '../../styles/inpage-hgroup';
+import { Fold } from '../../styles/fold';
+import MediaImage from '../../styles/media-image';
+import media from '../../styles/utils/media-queries';
+
+import Heading from '../../styles/type/heading';
+import config from '../../config';
+import { indicatorGroupColors } from '../../styles/theme/theme';
+import { IntroLead } from '../../styles/datasets';
+
+const { baseUrl } = config;
+
+const IntroFold = styled(Fold)`
+  padding-bottom: 0;
+
+  ${Prose} {
+    grid-column: content-start / content-end;
+
+    ${media.mediumUp`
+      grid-column: content-start / content-8;
+    `}
+
+    ${media.largeUp`
+      grid-column: content-start / content-10;
+    `}
+  }
+`;
+
+const ApproachFold = styled(Fold)`
+  padding-bottom: 0;
+
+  ${Gridder} {
+    align-items: center;
+  }
+
+  /* stylelint-disable-next-line */
+  ${InpageHGroup} {
+    grid-row: 1;
+    grid-column: content-start / content-end;
+
+    ${media.largeUp`
+      grid-column: content-start / content-7;
+    `}
+  }
+
+  ${Prose} {
+    grid-column: content-start / content-end;
+
+    ${media.mediumUp`
+      grid-column: content-start / content-8;
+    `}
+
+    ${media.largeUp`
+      grid-column: content-start / content-10;
+    `}
+  }
+`;
+
+const InterpretingDataFold = styled(Fold)`
+  padding-bottom: 0;
+
+  ${Prose} {
+    grid-column: content-start / content-end;
+
+    ${media.mediumUp`
+      grid-column: content-start / content-8;
+    `}
+
+    ${media.largeUp`
+      grid-column: content-start / content-10;
+    `}
+  }
+`;
+
+const CreditsFold = styled(Fold)`
+  padding-bottom: 0;
+
+  ${Gridder} {
+    align-items: center;
+  }
+
+  /* stylelint-disable-next-line */
+  ${InpageHGroup} {
+    grid-row: 1;
+    grid-column: content-start / content-end;
+
+    ${media.largeUp`
+      grid-column: content-start / content-7;
+    `}
+  }
+
+  ${Prose} {
+    grid-column: content-start / content-end;
+
+    ${media.mediumUp`
+      grid-column: content-start / content-8;
+    `}
+
+    ${media.largeUp`
+      grid-column: content-start / content-10;
+    `}
+  }
+`;
+
+const AdditionalResourcesFold = styled(Fold)`
+  ${Gridder} {
+    align-items: center;
+  }
+
+  /* stylelint-disable-next-line */
+  ${InpageHGroup} {
+    grid-row: 1;
+    grid-column: content-start / content-end;
+
+    ${media.largeUp`
+      grid-column: content-start / content-7;
+    `}
+  }
+
+  ${Prose} {
+    grid-column: content-start / content-end;
+
+    ${media.mediumUp`
+      grid-column: content-start / content-8;
+    `}
+
+    ${media.largeUp`
+      grid-column: content-start / content-10;
+    `}
+  }
+`;
+
+const metadata = {
+  id: 'shipping',
+  name: 'Shipping',
+  color: indicatorGroupColors.economic.color
+};
+
+class WQLongForm extends React.Component {
```

Comment:
Rename to match indicator

Reply:
I believe shipping is part of "Economic" indicator group, see https://github.com/NASA-IMPACT/covid-dashboard/issues/217.

Reply:
@danielfdsilva was referring to the name of the component `WQLongForm` (WQ = water quality), not the name of the indicator group. You're right that shipping is part of the `economic` groulp.